src/commandline_frame.ts

```diff
@@ -147,10 +149,13 @@ clInput.addEventListener("keydown", function (keyevent) {
 })
 
 clInput.addEventListener("input", () => {
+    const exstr = clInput.value
+    let expandedCmd = aliases.expandExstr(exstr)
```

Comment:
You could cut down on the number of variables here, if you don't feel like it you should at least make expandedCmd a const to match exstr.

`const exstr = aliases.expandExstr(clInput.value)`